[2021-06-27 07:16:53,554] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:16:24.179692+00:00 [queued]>
[2021-06-27 07:16:53,610] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:16:24.179692+00:00 [queued]>
[2021-06-27 07:16:53,614] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:16:53,620] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 07:16:53,634] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:16:53,656] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_postgresql_invoice> on 2021-06-27T07:16:24.179692+00:00
[2021-06-27 07:16:53,680] {standard_task_runner.py:52} INFO - Started process 5104 to run task
[2021-06-27 07:16:53,694] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_postgresql_invoice', '2021-06-27T07:16:24.179692+00:00', '--job-id', '45', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmp7vt7qvfz', '--error-file', '/tmp/tmptm96gq0g']
[2021-06-27 07:16:53,700] {standard_task_runner.py:77} INFO - Job 45: Subtask export_postgresql_invoice
[2021-06-27 07:16:53,966] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:16:24.179692+00:00 [running]> on host b69deda3a989
[2021-06-27 07:16:54,167] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_postgresql_invoice
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T07:16:24.179692+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T07:16:24.179692+00:00
[2021-06-27 07:16:54,179] {logging_mixin.py:104} INFO - Entering export_postgresql_to_csv SELECT * FROM test.invoice
[2021-06-27 07:16:54,223] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:17:02,230] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:17:02,282] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 07:17:02,304] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_postgresql_invoice, execution_date=20210627T071624, start_date=20210627T071653, end_date=20210627T071702
[2021-06-27 07:17:02,394] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-27 07:17:02,488] {local_task_job.py:151} INFO - Task exited with return code 0
